global.html

```diff
@@ -275,35 +275,26 @@ <h1>Welcome to R<sub style="margin-right: 0.1em;">t</sub>live global</h1>
         </div>
         <div id="rankingChart" class="svg-container"></div>
       </div>
-    </section>
+    </section> -->
 
     <section class="container">
-      <!--<div class="bs-callout bs-callout-warning">
-        <h5>Attention</h5>
-        <p>
-          Austria did <a href="https://www.reuters.com/article/us-health-coronavirus-austria-idUSKBN29E0HL">large scale testing of school pupils</a>
-          This
-          <a href="https://ourworldindata.org/coronavirus-data-explorer?zoomToSelection=true&time=2020-03-01..latest&country=DEU~FRA~AUT&region=World&testsMetric=true&interval=smoothed&smoothing=7&pickerMetric=location&pickerSort=asc">
-            distorts the data
-          </a> and prediction of daily performed tests.
-          If these distortions do not stabilize, we'll have to remove Austria from the site.
-        </p>
-      </div>-->
       <div class="bs-callout bs-callout-warning">
-        <h5>Attention</h5>
+        <h5>Information</h5>
         <p>
-          As you can see we are running the analysis on a large number of countries/regions.
-          To ensure a high quality of the results we depend on local domain expertise.
+          For about a year we provided daily R-value nowcasting for Germany and further countries on this page.
           </br>
-          If you notice data quality issues (e.g. outliers), please file a bug report at <a href="https://github.com/rtcovidlive/rtlive-global">github.com/rtcovidlive/rtlive-global</a>
+          Lately the data and the model don't align  with each other and because we don't have enough time to continue maintaining the model, we decided to take it offline.
```

Comment:
```suggestion
          Lately the data and the model don't align with each other and because we don't have enough resources to continue maintaining the model, we decided to take it offline.
```